group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroup.java

```diff
@@ -288,14 +291,15 @@ public ConsumerGroupMember getOrMaybeCreateMember(
     /**
      * Updates the member.
      *
+     * @param topicsImage The current metadata for all available topics.
```

Comment:
The order of parameters must be inverted i.e `newMember` must be specified first and then `topicsImage`